### PR TITLE
[CGMES GL] Add SubstationPosition extension when location on voltage levels

### PR DESCRIPTION
**Pull request by flo-dup**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**
No

**What kind of change does this PR introduce?**
feature

**What is the current behavior?**
In CGMES GL import, only locations on substations PowerSystemResource are supported

**What is the new behavior (if this is a feature change)?**
In CGMES GL import, locations on substations and voltage levels PowerSystemResource are supported

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

**Other information**:
- When several voltage level locations inside the same substation, the average latitude and longitude is computed
- When voltage level location inside a substation with location, the voltage level location is dismissed

### CODE REVIEW

File: cgmes/cgmes-gl/src/test/resources/importGL_EQ.xml [hunk 1]
```diff
@@ -35,11 +39,22 @@
     <cim:VoltageLevel.Substation rdf:resource="#_Substation2"/>
     <cim:IdentifiedObject.name>Voltage level 1</cim:IdentifiedObject.name>
   </cim:VoltageLevel>
+  <cim:VoltageLevel rdf:ID="_VoltageLevel3">
+    <cim:VoltageLevel.BaseVoltage rdf:resource="#_BaseVoltage300"/>
+    <cim:VoltageLevel.Substation rdf:resource="#_Substation3"/>
+    <cim:IdentifiedObject.name>Voltage level 1</cim:IdentifiedObject.name>
```
Comment (olperr1) [inline cgmes/cgmes-gl/src/test/resources/importGL_EQ.xml:45]:
```suggestion
    <cim:IdentifiedObject.name>Voltage level 3</cim:IdentifiedObject.name>
```

File: cgmes/cgmes-gl/src/test/resources/importGL_EQ.xml [hunk 2]
```diff
@@ -35,11 +39,22 @@
...
+  </cim:VoltageLevel>
+  <cim:VoltageLevel rdf:ID="_VoltageLevel4">
+    <cim:VoltageLevel.BaseVoltage rdf:resource="#_BaseVoltage300"/>
+    <cim:VoltageLevel.Substation rdf:resource="#_Substation3"/>
+    <cim:IdentifiedObject.name>Voltage level 1</cim:IdentifiedObject.name>
```
Comment (olperr1) [inline cgmes/cgmes-gl/src/test/resources/importGL_EQ.xml:50]:
```suggestion
    <cim:IdentifiedObject.name>Voltage level 4</cim:IdentifiedObject.name>
```

**Review by olperr1 — APPROVED** [review]

File: cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java [hunk 3]
```diff
@@ -49,9 +49,9 @@ private PropertyBags queryTripleStore(String queryKey) {
         return tripleStore.query(query);
     }
 
-    public PropertyBags getSubstationsPosition() {
-        LOG.info("Querying triple store for substations positions");
-        return queryTripleStore(SUBSTASTION_POSITION_QUERY_KEY);
+    public PropertyBags getSubstationsVoltageLevelsPosition() {
```
Comment (So-Fras) [inline cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java:52]:
```suggestion
    public PropertyBags getSubstationVoltageLevelPosition() {
```

File: cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java [hunk 4]
```diff
@@ -49,9 +49,9 @@ private PropertyBags queryTripleStore(String queryKey) {
...
+        LOG.info("Querying triple store for substations and voltage levels positions");
```
Comment (So-Fras) [inline cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java:53]:
```suggestion
        LOG.info("Querying triple store for substation and voltage level positions");
```

File: cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java [hunk 5]
```diff
@@ -22,7 +22,7 @@
  */
 public class CgmesGLModel {
 
-    public static final String SUBSTASTION_POSITION_QUERY_KEY = "substationPosition";
+    public static final String SUBSTATION_VL_POSITIONS_QUERY_KEY = "substationVoltageLevelPosition";
```
Comment (So-Fras) [inline cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java:25]:
```suggestion
    public static final String SUBSTATION_VL_POSITION_QUERY_KEY = "substationVoltageLevelPosition";
```

File: cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java [hunk 6]
```diff
@@ -49,9 +49,9 @@ private PropertyBags queryTripleStore(String queryKey) {
         return tripleStore.query(query);
     }
 
-    public PropertyBags getSubstationsPosition() {
-        LOG.info("Querying triple store for substations positions");
-        return queryTripleStore(SUBSTASTION_POSITION_QUERY_KEY);
+    public PropertyBags getSubstationsVoltageLevelsPosition() {
+        LOG.info("Querying triple store for substations and voltage levels positions");
+        return queryTripleStore(SUBSTATION_VL_POSITIONS_QUERY_KEY);
```
Comment (So-Fras) [inline cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java:54]:
```suggestion
        return queryTripleStore(SUBSTATION_VL_POSITION_QUERY_KEY);
```

**Review by So-Fras — CHANGES_REQUESTED** [review]

**Review by So-Fras — APPROVED** [review]

File: cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java (hunk 3 above)
Comment (flo-dup) [reply cgmes/cgmes-gl/src/main/java/com/powsybl/cgmes/gl/CgmesGLModel.java]: `getSubstationVoltageLevelPositions()`

**Pull request merged**